chapter06/red/exercise1.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $db = '.coconet.db';
+my %stats;
+%stats = %{ retrieve $db } if -f $db;
+
+my $fn = shift @ARGV // 'coconet.dat';
+open my $fh, '<', $fn;
+
+while( <$fh> ) {
+  next if /^\s*(#.*)?$/;
```

Comment:
I tried editing using GitHub's editor to quickly preview what it might be, and the syntax highlighting was correct there :angry:.